src/libloader/CMakeLists.txt

```diff
@@ -5,5 +5,27 @@ add_sources (elektra-full "static.c")
 if (BUILD_SHARED)
 	add_sources (elektra-shared "dl.c")
 	add_includes (elektra-shared)
+
+	if (${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
+		set(BUILD_ARGS -DCMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}
+			-DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>
+			-DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
+		)
+
+		if (CMAKE_TOOLCHAIN_FILE)
+			list(APPEND BUILD_ARGS -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE})
```

Comment:
That sounds strange to me. Why append another CMAKE_TOOLCHAIN_FILE if the user already supplied one?